examples/java-agent/pom.xml

```diff
@@ -0,0 +1,100 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>example</groupId>
+  <artifactId>helloworld</artifactId>
+  <version>1</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jib-maven-plugin.version>0.9.11</jib-maven-plugin.version>
+    <maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
+    <download-maven-plugin.version>1.4.1</download-maven-plugin.version>
+
+    <!-- agents will be extracted relative to this build location -->
+    <agent-extraction-root>${project.build.directory}/agents</agent-extraction-root>
+    
+    <!-- location of Stackdriver Debugger Java Agent https://cloud.google.com/debugger/docs/setup/java -->
+    <stackdriver-debugger-agent-url>https://storage.googleapis.com/cloud-debugger/compute-java/debian-wheezy/cdbg_java_agent_gce.tar.gz</stackdriver-debugger-agent-url>
+
+    <!-- where to place the Stackdriver Debugger Java Agent in the container -->
+    <stackdriver-debugger-agent-location>/opt/cdbg</stackdriver-debugger-agent-location>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>23.6-jre</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler-plugin.version}</version>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
+
+      <!-- Jib -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>${jib-maven-plugin.version}</version>
+        <configuration>
+          <to>
+            <image>gcr.io/REPLACE-WITH-YOUR-GCP-PROJECT/image-built-with-jib</image>
+          </to>
+          <extraDirectory>${agent-extraction-root}</extraDirectory>
+          <container>
+            <jvmFlags>
+              <jvmFlag>-agentpath:${stackdriver-debugger-agent-location}/cdbg_java_agent.so</jvmFlag>
+              <jvmFlag>-Dcom.google.cdbg.module=${project.artifactId}</jvmFlag>
+              <jvmFlag>-Dcom.google.cdbg.version=${project.version}</jvmFlag>
+              <!--
+                 if not running on Compute Engine, must download and copy a service key
```

Comment:
let's not encourage users copying service keys into a container image. use secrets instead.

Reply:
Suggestions welcome!

Reply:
I've dropped this text and will rely on the pointers in README.md instead.